src/libraries/Directory.Build.props

```diff
@@ -17,7 +17,7 @@
     <IsReferenceAssembly Condition="'$(IsReferenceAssembly)' == '' and ($(MSBuildProjectFullPath.Contains('\ref\')) or $(MSBuildProjectFullPath.Contains('/ref/')))">true</IsReferenceAssembly> 
     <LanguageTargets Condition="'$(MSBuildProjectExtension)' == '.depproj'">$(RepositoryEngineeringDir)depProj.common.targets</LanguageTargets>
     <RuntimeGraph>$(LibrariesProjectRoot)OSGroups.json</RuntimeGraph>
-    <BuildTargetFramework>netcoreapp5.0</BuildTargetFramework>
+    <BuildTargetFramework>$(NetCoreAppCurrent)</BuildTargetFramework>
```

Comment:
Should we change the variable name $(NetCoreAppCurrent) to something else as its no longer netcoreapp

Reply:
No, the TargetFrameworkIdentifier is still NetCoreApp. Not planning to do this as we have **many** places throughout our codebase that refer to .NETCoreApp.